### ISSUE TITLE
No notification on closing / repeated closing and opening to the same side

### ISSUE THREAD

**Issue by merelj** [letsar/flutter_slidable#226]
Hey there again 👋 
Thanks a lot for your help with https://github.com/letsar/flutter_slidable/issues/225

I've found another small issue though: when I subscribe to actionPaneType ValueNotifier its value never becomes equal to ActionPaneType.none after closing gesture -- and, therefore, no events are received when I open and close it to the same side multiple times in a row.

It's certainly not a big problem, but I thought I should mention it anyway.

my code:
```
  late final ValueNotifier<ActionPaneType>? _actionPaneTypeValueNotifier;
  @override
  void initState() { 
    super.initState();
    _actionPaneTypeValueNotifier = Slidable.of(context)?.actionPaneType;
    _actionPaneTypeValueNotifier?.addListener(_onActionPaneTypeChanged);
  }

  @override
  void dispose() {
    _actionPaneTypeValueNotifier?.removeListener(_onActionPaneTypeChanged);
    super.dispose();
  }

  void _onActionPaneTypeChanged() {
    print('Value is ${_actionPaneTypeValueNotifier?.value}') // I receive ActionPaneType.start 1 time and after that nothing (unless I get ActionPaneType.end, after which ActionPaneType.start starts working, but .end stops).
  }
```

**Comment by letsar** [comment]
Hi @merelj ! Thank you for your feedback, I see where it comes from, I'll fix it.

**Comment by letsar** [comment]
Fixed in 1.0.0-dev.6
